src/repart/repart.c

```diff
@@ -4312,6 +4313,9 @@ static int context_wipe_range(Context *context, uint64_t offset, uint64_t size)
         }
 
         return 0;
+#else
+        return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "libblkid support is not compiled in.");
```

Comment:
can you say what fails here, i.e. "Cannot wipe partition signatures, libblkid support is not compiled in." or so?

Reply:
Sure, done.